jobserver/templates/project_membership_edit.html

```diff
@@ -94,6 +94,22 @@ <h4>Roles</h4>
       </form>
     </div>
 
+    <hr />
+
+    <div>
+      <h4>Remove Member</h4>
+
+      <form method="POST" action="{{ membership.get_remove_url }}">
```

Comment:
Yep, this was moved from the settings template to this one since I thought membership removal would likely be better suited in the edit page.  Not a strongly held opinion though, so happy to be disuaded of the move if you do feel strongly!